### DISCUSSION TITLE
Clearance rate of different doses of the same drug

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1279] [Questions & Problems]
When modeling, using the same compound parameters for different doses of the same drug, one dose is found to fit well and the other dose is found to fit poorly. Is there a difference in clearance rates among different doses?

**Comment by StephanSchaller** [comment]
Well, this can only be answered by knowing how the clearance (and possibly other active processes) is set up in the model.
SO if you can elaborate on this, someone here might be able to help.

BEst,
Stephan

**Reply by WTY-tt** [reply]
![代谢](https://user-images.githubusercontent.com/95362728/185005833-3162beb5-4cee-4748-891f-b66a76838073.jpg)
![代谢2](https://user-images.githubusercontent.com/95362728/185005849-3addc672-f7b8-433e-900e-eb0937089f18.jpg)
